src/dimensions.ts

```diff
@@ -106,6 +106,18 @@ export const VirtualSizes = Object.freeze({
   // For N inputs, the overestimation will be below N vbytes for all transactions.
   txP2wshInputSize: 105,
 
+  // TODO: Calculate script path spend sizes for p2tr inputs
+  // Keypath spend size is calculated by adding:
+  //   - 36 vbyte outpoint
+  //   - 1 vbyte scriptSig
+  //   - 4 vbyte nSequence
+  //   - 1 WU witness item count
+  //   - 1 WU witness item size
+  //   - 64 WU schnorr signature
+  // 32+4+1+4+(1+1+64)/4 = 57.5 vB - rounded up to 58
+  // See: https://murchandamus.medium.com/2-of-3-multisig-inputs-using-pay-to-taproot-d5faf2312ba3
+  txP2trInputSize: 58,
```

Comment:
we should call this `txP2trKeypathInputSize`

Reply:
Yeah that makes sense, this is something of a placeholder pending the script path spend logic but might as well call it this now. I'm thinking that eventually we should have this, a size for a first level taptree script spend, and a size for a second level taptree script spend.